GameBoard.py

```diff
@@ -1,22 +1,24 @@
+from typing import List
```

Comment:
Is this mandatory ?

Reply:
Yup. `list` is a keyword that does not fall in the scope of type hints. However, the `List` class from `typing` does.